lib/phoenix_live_view.ex

```diff
@@ -1864,7 +1864,13 @@ defmodule Phoenix.LiveView do
   As shown, an existing item on the client can be updated by issuing a `stream_insert`
   for the existing item. When the client updates an existing item, the item will remain
   in the same location as it was previously, and will not be moved to the end of the
-  parent children. To both update an existing item and move it to another position,
+  parent children. 
+
+  One exception to this is if you call `stream` with a collection of items in your `mount` function, 
+  and then `stream_insert` an updated item that was previously streamed in your `handle_params` function. 
+  You will end up with a duplicate item. 
```

Comment:
```suggestion
  > #### A note on duplicates, streams and the LiveView life-cycle {: .info}
  > LiveView does not support updating an already streamed item during in same render cycle.
  > This can affect you in the following ways:
  >
  > * If your list of items passed to `stream/4` already contains duplicates,
  >   these duplicates will end up on the page. You'll need to deduplicate them before calling `stream/4`.
  > * Due to the life-cycle of a LiveView, if you call `stream/4` with a list of items in
  >   your `c:mount/3` callback, you cannot use `stream_insert/4` in `c:handle_params/3`
  >   to update an item that was already inserted in `c:mount/3`. As there is no render in between
  >   those two callbacks on the initial mount, you would end up with a duplicate item in the stream.
  >   You can work-around this limitation by setting an "initial_mounted" assign in `c:mount/3`
  >   and pattern-match on this flag in `c:handle_params/3` to handle the different cases.
```

cc @josevalim